src/styles/card.scss

```diff
@@ -0,0 +1,153 @@
+@import './variables';
+
+.card {
+  box-sizing: border-box;
+  align-items: center;
+  width: 200px;
+  background-color: white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+  transition: transform 300ms ease-in-out;
+
+  &:hover {
+    transform: scale(1.2); // Збільшення карти на 20%
+  }
+
+  &__image {
+    box-sizing: border-box;
+    background-image: url(../images/imac.jpeg);
+    width: 160px;
+    height: 134px;
+    background-size: cover;
+    margin: 0 auto;
+  }
+
+  &__product-name {
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: #{$main-accent};
+    width: 100%;
+    margin-top: 40px;
+    margin-bottom: 0;
+  }
+
+  &__product-id {
+    font-family: Roboto, sans-serif;
```

Comment:
The font family is already defined in the `body` in the `index.scss` file. So, there is no need to define it again here. Please remove this line.